protocol/signalfx/format/log/signalfx_log.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+package sfx_log_format;
+
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "gen_go/;log_format";
+
+// model is based off https://github.com/open-telemetry/oteps/blob/master/text/0097-log-data-model.md#log-and-event-record-definition
+message LogRecord {
+    google.protobuf.Timestamp Timestamp = 1;
+    string TraceID = 2;
+    string SpanID = 3;
+    string TraceFlags = 4;
```

Comment:
TraceFlags are a byte in Otel log data model.

Reply:
according to [https://developers.google.com/protocol-buffers/docs/proto3](url) there is not single byte field so I think i should use bytes

Reply:
`bytes` will be also slow. I would just use an fixed32 or int32. It is fast and compact.

Reply:
added fixed32